store/items.js

```diff
@@ -74,11 +78,13 @@ export const getters = {
       getters.counts.chars * costPerChar +
       getters.counts.quotes * costPerQuote +
       getters.counts.sounds * costPerSound
-    if (differenceInHours(getters.deadline, new Date()) > 48) {
+
+    // TODO: serious race condition here! Open for suggestions!
+    if (differenceInHours(getters.deadline, new Date()) < 48) {
```

Comment:
Small bug here? `>` instead of `<`?

Reply:
Nein das ist so korrekt. Nur wenn die timerange länger als 2 tage bzw 48 stunden ist, sollen die expressFee applied werden